Added info with Colab links into README

DmitriyStoyanov opened the pull request:
Thanks for your repo

For the moment, README of this project is clean, but there we can add some more info, with description of project and direct links that can be opened from start page of this repo

What do you think? should we add such info here?
If so, need to add more links for all other variants, currently added only 3 files

DmitriyStoyanov committed:
Added info with Colab links into README

dsinkerii reviewed (changes requested):
would be nice to see a list of all colab links, like in https://github.com/camenduru/stable-diffusion-webui-colab (like a table), but i agree with you, that would be nice to see a proper readme. also, would be great if all of the colab files were in a folder, so when you visit the repo, you could see the readme without scrolling down the page just to see it

DmitriyStoyanov committed:
Move all notebooks to separate folder and adding table into ReadMe

DmitriyStoyanov commented:
You mean something like that?
- Moved all notebooks into separate folder
- Added table with link to Collab, Info, Link to civitai model

https://github.com/DmitriyStoyanov/sd-1click-colab

DmitriyStoyanov committed:
Added urls for all notebooks

DmitriyStoyanov commented:
Added links for all notebooks in folder.
Now need to add information about each model used in the list.

Created list using next script:
```shell
for i in $(ls notebooks); do echo '[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/nolanaatama/sd-1click-colab/blob/main/notebooks/'$i') | '$i' |'; done
```
